tools/system_libs.py

```diff
@@ -264,6 +249,41 @@ def create_wasm_compiler_rt(libname):
     run_commands([[shared.LLVM_AR, 'cr', '-format=gnu', lib] + o_s])
     return lib
 
+  def files_in_path(path_components=None, filenames=None):
```

Comment:
overall looks ok to me, but one comment - could this method be used in the other `create*` methods too?